stubs/setupVars.conf

```diff
@@ -0,0 +1,3 @@
+piholeInterface=eth0
+IPv4_address=192.168.1.100/24
+IPv6_address=fe80::bc3b:7855:7c13:4f84
```

Comment:
I see, I grabbed these from my local copy of pi-hole which is still on master, I'll update.